frontend/src/components/sidebar/SideBar.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import Drawer from "react-modern-drawer";
+import "react-modern-drawer/dist/index.css";
+import { NavLink } from "react-router-dom";
+import classes from "./SideBar.module.scss";
+import { ReactComponent as SettingsIcon } from "../../assets/icons/settings-circle.svg";
+import { ReactComponent as ConnectIcon } from "../../assets/icons/connect-circle .svg";
+import { ReactComponent as SwitchIcon } from "../../assets/icons/switch.svg";
+import { ReactComponent as PlusIcon } from "../../assets/icons/plus.svg";
+import IconButton from "components/icon-button/IconButton";
+
+export type Sidebar = {
+  toggled: boolean;
+  toggleSidebar: () => void;
+};
+
+function SideBar({ toggled, toggleSidebar }: Sidebar) {
+  // TODO(sideBar-component): Check if it is possible (with currently used library "react-modern-drawer")
+  // to position sidebar bellow navbar instead of overlaping it
```

Comment:
I was thinking it might be possible to just "force" navigation to be displayed above the "Sidebar" component (in z direction). Maybe we could enforce that with `z-index` css property, but I'm not really sure.

The only thing that would be need then is to add some offset to the top of the sidebar component, so that the navigation doesn't overlap with the sidebar buttons.

Does that make sense? I think I can try that approach real quick, just to see if that could work. If that doesn't work, I think we should just leave it as it is.

Reply:
Hmm, I tried it out, but it looks kinda odd/confusing that way.

![Screenshot 2022-09-02 at 02 40 25](https://user-images.githubusercontent.com/36109955/188035192-41548787-a683-433f-863f-350a05bf8d4f.png)

I think we should just leave it as it was before and maybe even hide the log drawer bellow it.

Reply:
I think we can safely remove this comment now.

Reply:
Nice!